lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -103,10 +103,10 @@ SmallVector<unsigned> getSizePerThread(Attribute layout) {
     return SmallVector<unsigned>(blockedLayout.getSizePerThread().begin(),
                                  blockedLayout.getSizePerThread().end());
   } else if (auto sliceLayout = layout.dyn_cast<SliceEncodingAttr>()) {
-    auto ret = getSizePerThread(sliceLayout.getParent());
-    return ret;
-    // ret.erase(ret.begin() + sliceLayout.getDim());
-    return ret;
+    // TODO: maybe should not be supported
```

Comment:
I don't understand this TODO :) Would it be possible to figure it out before we merge this PR?

Reply:
Ah, forgot to remove the comment, I thought the linter would have caught this.